students/seunghun/users/models.py

```diff
@@ -0,0 +1,22 @@
+from django.db import models
+from django.db.models.deletion import CASCADE
+
+# Create your models here.
+
+class Essential_User(models.Model):
+    name = models.CharField(max_length=10)
+    email = models.CharField(max_length=45)
+    password = models.CharField(max_length=20)
+    phone_num = models.CharField(max_length=20)
```

Comment:
변수명은 줄이지 않고 명확하게 작성해주는 것이 좋습니다.
```suggestion
    phone_number = models.CharField(max_length=20)
```

Reply:
넵 수정하겟습니다!